redfish-core/lib/account_service.hpp

```diff
@@ -2147,6 +2194,16 @@ inline void requestAccountServiceRoutes(App& app)
                 const std::string userPath =
                     "/xyz/openbmc_project/user/" + username;
 
+                // Don't DELETE accounts which have a Restricted Role (the service account).
+                // Implementation note: Ideally this would get the user's RoleId
+                // but that would take an additional D-Bus operation.
+                if (username == "service")
```

Comment:
I am struggling a bit with the hardcoding and no D-Bus call..
I guess..

Reply:
We ran out of time to do this properly.  Phosphor-user-manager should handle the Restricted Role, then bmcweb can generalize this code.
The "service" account really is a built-in account which has special handling.